RoboFile.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+include_once dirname(__FILE__) . '/vendor/autoload.php';
```

Comment:
Shouldn't do this. Do you have this in here because you are using a separate Robo.phar to run the Robofile, rather than using the Robo inside of Terminus? I think the best solution is to not make the Robofile call Terminus code.

Reply:
It looks like the Robofile does not call Terminus code (except via exec-ing the .phar), so you should be able to just remove this line. n.b. keeping it risks hard-to-diagnose crashes.